lib/config.go

```diff
@@ -182,6 +182,12 @@ type RuntimeConfig struct {
 
 	// BindMountPrefix is the prefix to use for the source of the bind mounts.
 	BindMountPrefix string `toml:"bind_mount_prefix"`
+
+	// SubUIDName is the name to lookup in /etc/sub_uid for setting up the userNS.
+	SubUIDName string `toml:"sub_uid_name"`
```

Comment:
I think we could just work with daemon-level UIDMappings/GIDMappings for cri-o. I don't see us creating a user for each user namespace when we get to multiple user namespaces in k8s. k8s wouldn't care about this detail as it is too low-level and  best left to the runtime. WDYT?

Reply:
yes I think the UID-GID mappings must be passed by k8s down to CRI-O as a global setting doesn't make much sense.

I was mostly looking for a way to plug the support for userNS somehow, so I can start testing it.  I thought about having a "default" userNS setting, that is used by default for all containers if it is not specified by k8s.

Reply:
@giuseppe For phase 1, the plan is for the CRI runtimes to use their own global UID/GID mappings and return those to kube when requested over a new CRI call. I think instead of SubUIDName/SubGIDName, we should have a global field for UIDMappings/GIDMappings in anticipation of kube phase 1.